src/domain/index.ts

```diff
@@ -39,13 +40,15 @@ export function getTransactionStatusText(status: TransactionStatus): string {
 }
 
 export enum Currency {
-  EUR = "eur",
-  USD = "usd",
-  JPY = "jpy",
-  GBP = "gbp",
-  CNY = "cny",
+  EUR = "EUR",
+  USD = "USD",
+  JPY = "JPY",
+  GBP = "GBP",
+  CNY = "CNY",
 }
 
+export type FiatExchangeRates = Record<string, number>;
```

Comment:
I'd declare this as `Record<keyof typeof Currency, number>` so the resulting type is:

```typescript
type FiatExchangeRates = {
  EUR: number;
  USD: number;
  JPY: number;
  GBP: number;
  CNY: number;
}
```

Reply:
Done!